docs/docs/kb/03-config.md

```diff
@@ -4,187 +4,260 @@ description: Primary configuration file to describe the development environment
+If you want an account to be initialized from a specific mnemonic, provide the
+`mnemonic` field with a valid mnemonic. A private key, a public key and an
+address will be derived from a mnemonic.
 
-```yaml
-build:
-  binary: "mychaind"
-  ldflags: [ "-X main.Version=development", "-X main.Date=01/05/2022T19:54" ]
+```yml
+accounts:
+  - name: bob
+    coins: ['20000token', '200000000stake']
+    mnemonic: cargo ramp supreme review change various throw air figure humble soft steel slam pole betray inhale already dentist enough away office apple sample glue
 ```
 
-### build.proto
-
-| Key               | Required | Type            | Description                                                                                  |
-|-------------------|----------|-----------------|----------------------------------------------------------------------------------------------|
-| path              | N        | String          | Path to protocol buffer files. Default: `"proto"`.                                           |
-| third_party_paths | N        | List of Strings | Path to third-party protocol buffer files. Default: `["third_party/proto", "proto_vendor"]`. |
+You cannot have both `address` and `mnemonic` defined for a single account.
 
-## client
+Some accounts are used as validator accounts (see `validators` section).
+Validator accounts cannot have an `address` field, because Ignite needs to be
+able to derive a private key (either from a random mnemonic or from a specific
+one provided in the `mnemonic` field). Validator accounts should have enough
+tokens of the staking denomination for self-delegation.
 
-Configures client code generation.
+By default, the `alice` account is used as a validator account, its key is
+derived from a mnemonic generated randomly at genesis, the staking denomination
+is `stake`, and this account has enough `stake` for self-delegation.
 
-### client.vuex
+If your chain is using its own
+[cointype](https://github.com/satoshilabs/slips/blob/master/slip-0044.md), you
+can use the `cointype` field to provide the integer value
 
-```yaml
-client:
-  vuex:
-    path: "vue/src/store"
+```yml
+accounts:
+  - name: bob
+    coins: ['20000token', '200000000stake']
+    cointype: 7777777
 ```
 
-Generates Vuex stores for the blockchain in `path` on `serve` and `build` commands.
+## Validators
 
-### client.typescript
+Commands like `ignite chain init` and `ignite chain serve` initialize and launch
+a validator node for development purposes.
 
-```yaml
-client:
-  typescript:
-    path: "vue/src/generated"
+```yml
+validators:
+  - name: alice
+    bonded: '100000000stake'
 ```
 
-Generates TypeScript clients for the blockchain in `path` on `serve` and `build` commands.
+`name` refers to key name in the `accounts` list.
 
-### client.openapi
+`bonded` is the self-delegation amount of a validator. The `bonded` amount
+should not be lower than `1000000` and should be higher than the account's
+balance in the `account` list.
```

Comment:
I am not fully sure about this one but I wanted to point out that might need to be corrected.

```suggestion
should not be lower than `1000000` nor higher than the account's
balance in the `account` list.
```

Reply:
Good catch!